doc/ref_arch/openstack/chapters/chapter01.rst

```diff
@@ -147,10 +147,51 @@ to address them (for example, resources autoscaling).
 Terminology
 -----------
 
-General terminology definitions can be found in
-:doc:`cntt:common/glossary` and specific terms relating
-to this reference architecture are to be found in `OpenStack Related
-Terminology :ref:`cntt:common/glossary:openstack related terminology`.
+.. glossary::
+
+    Cloud Infrastructure
+      A generic term covering **NFVI**, **IaaS** and **CaaS** capabilities -
+      essentially the infrastructure on which a **Workload** can be executed.
+
+.. note::
+
+  `The official OpenStack Glossary <https://docs.openstack.org/image-guide/common/glossary.html>`__
+  is an extensive list of OpenStack-related concepts. Some additional terms
+  used in the Reference Architecture RA-1 or used to relate RA-1 terms with
```

Comment:
"Some additional terms used in the Reference Architecture RA-1 or used to relate RA-1 terms with terms defined elsewhere."

is this trying to say, "This Glossary will only define additional terms used in the Reference Architecture RA-1 or used to relate RA-1 terms with terms defined elsewhere."

Reply:
I didn't change the content. I only touched the form.

Reply:
For my understanding, there will be one glossary per stream.
But if it helps maintenance we could host 2 glossaries in RA1, including one sync with RM.